manifests/n/Notion/Notion/2.0.32/Notion.Notion.installer.yaml

```
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.2.0.schema.json
+
+PackageIdentifier: Notion.Notion
+PackageVersion: 2.0.32
+Platform:
+- Windows.Desktop
+MinimumOSVersion: 10.0.0.0
+Scope: user
+InstallModes:
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x86

```

```suggestion
- Architecture: x64
```
Application only works on x64 OS.